lib/reporter/results.js

```diff
@@ -446,6 +446,11 @@ module.exports = class Results {
   }
 
   logCommand(command) {
+    // Masking passwords so they don't appear as plain text in reports.
+    if (command.name === 'setPassword') {
+      const newArgs = [command.args[0], '*'.repeat(command.args[1].length)];
+      command.args = newArgs;
+    }
```

Comment:
We shouldn't add fix for a very specific case in a general method like this, it bloats these methods. We should always try to make our solution general so that we can easily expand it to other places as well (think if more commands were to use the redaction functionality, how the solution we're adding could help them as well without hard coding the function names).

See here how this functionality was approached earlier and try to make changes along the same lines and using the existing code: https://github.com/nightwatchjs/nightwatch/pull/2672

The above solution no longer works because while earlier we used to call `TransportActions.post()` to send HTTP requests directly, we now use Selenium methods (`element.sendKeys()` here) to do so.

Reply:
Yes, I am looking into this only.

Reply:
Yeah sure, I just posted a message there so that other people can also try and find a solution to this, if they have no other issues to work on.